### ISSUE TITLE
is there support for cloud urls which have read access?

### ISSUE THREAD

**Issue by waqas977** [scottwernervt/cloudstorage#69]
Can we access a file in a bucket with file url with read access.Without providing credentials for that cloud ?

**Comment by scottwernervt** [comment]
I do not think this is possible as it looks like each driver backend requires credentials.
